src/bits/bit_array.rs

```diff
@@ -117,6 +140,27 @@ macro_rules! bit_array_impl {
                 }
             }
 
+            /// Compares two `BitArray`s by their representational ordering
+            ///
+            /// The original implementation of `Ord` for `bitvec::BitArray` compares two arrays
+            /// from LSB, and at the first index where the arrays differ, the array with the high
+            /// bit is greater. For our use case, however, we want to compare two arrays by their
+            /// integer values represented by the bits. In other words, if `a < b` is true, then
+            /// `BitArray::try_from(a).unwrap() < BitArray::try_from(b).unwrap()` must also be true.
+            impl Ord for $name {
```

Comment:
do we ever use bit arrays that store more than 128 bits? if we do, panicking in `Ord` is not the best practice imo.
It does not seem that we allow constructing bitarrays from anything expect a `u128`, so maybe we can define `TryFrom/Into` for u128 instead of `TryFrom/TryInto` and getting rid of `unwrap` here?

Reply:
Sounds good. I think we are okay with constraining to <= 128 bits for now.